### PR TITLE
Ux lewis and ed

### PR DESCRIPTION
**Pull request by Gleoman**
Picture functionality now included on homepage and after making a new listing.

When making a new listing the user is asked to upload a picture url/jpeg to the database which stores it as a property (string) on the space object.

When the spaces are then iterated over to produce a view, the image is rendered alongside other details of the listing.

### CODE REVIEW

File: app/app.rb [hunk 1]
```diff
@@ -43,14 +45,15 @@ class DwellBNB < Sinatra::Base
...
-                price: params[:price])
+                price: params[:price], picture_url: params[:picture_url])
     add_availability(space)
     space.save
     redirect '/spaces'
   end
 
   get '/spaces' do
     @listings = Space.all
+		# binding.pry
```
Comment (alextwilson) [inline app/app.rb]: Can we get rid of this binding?

Comment (alextwilson) [reply app/app.rb]: Apart from this, :shipit:

**Review by ghost — COMMENTED** [review]
👍

**Comment by lewmoore** [comment]
:shipit: (without the binding.pry)

**Comment by alextwilson** [comment]
:shipit:

**Comment by Gleoman** [comment]
:shipit:

**Pull request merged**